### ISSUE TITLE
Vulnerability report has found zero vulnerabilities

### ISSUE THREAD

**Issue by sebw** [stackrox/stackrox#1244]
I'm running a ~daily~ weeky report at 2am and send by email.

There are 2 problems:

- the logo is Red Hat Advanced Security for Kubernetes
- the report says there are no found vulnerabilities

**Comment by Maddosaurus** [comment]
Hi @sebw and thank you for filing that issue!
The report logo is a great catch, let me relay this to the appropriate team!

To assist in tracking down the other issue, it would be great if you could provide us a bit of context:
- Do you have active new violations that the report should include?
- What version of StackRox are you running? Have you deployed from master or pulled Docker images?
- Which version (if applicable) of StackRox are you running?
- On what Kubernetes flavor have you deployed StackRox? (e.g. vanilla k8s, Docker for Desktop, EKS, GKE, OKD, ...)

**Comment by sebw** [comment]
Hi @Maddosaurus 

I run `3.69.x-nightly-20220402` on k3s `v1.22.5+k3s1`. I know k3s is not supported but a quick hack to one of the install script and I could get it installed, and besides reporting everything works as expected!

There are a couple of new violations that poped up I think, but I run a report that is supposed to show all vulnerabilities, and not only the new ones.

This is the config:

![image](https://user-images.githubusercontent.com/2285094/162776479-bc0c1ddc-76b3-40fa-a8cf-569e838c813b.png)

Let me know if you need more info.

**Comment by Maddosaurus** [comment]
Hey @sebw!
I've tried reproducing the reporting behaviour but was not able to produce an empty report.  
I'm running Docker Desktop 4.6.1 on OSX right now.

Exploring the possible differences between our configs, I'd like to ask you for a bit more information:
- Did the empty report show up on planned report execution, on "run report now", or on both?
- Could you attach a screenshot of your resource scope settings?

I have a hunch that this has something to to with the scoping.
Also, does this problem persist with a more recent version? (If you happen to have updated your deployment in the meantime)

**Comment by sebw** [comment]
Hi,

Running a report manually returns zero issues as well.

Attached the scope settings:

![image](https://user-images.githubusercontent.com/2285094/167956847-58c431ef-d673-4629-89de-ab8211f3b926.png)

So I figured the fact the cluster had a big red cross I had to edit it somehow. After allowing the cluster manually, the report contains a zip file with a csv:

![image](https://user-images.githubusercontent.com/2285094/167957322-91a1f003-a323-4e7d-8870-587481ca514a.png)

![image](https://user-images.githubusercontent.com/2285094/167957384-1703baa9-383b-4b80-91dc-26ecb024e031.png)

Is this expected the scope "all" excludes everything?
